google_photos_takeout_helper/__main__.py

```diff
@@ -254,7 +253,8 @@ def find_duplicates(path: Path, filter_fun=lambda file: True):
                 files_by_size[file_size].append(file)
 
         # For all files with the same file size, get their hash on the first 1024 bytes
-        for file_size, files in files_by_size.items():
+        logger.info('Step 1...')
+        for file_size, files in _tqdm(files_by_size.items()):
```

Comment:
The "step 1" is <0 seconds for me, so I left it - how long is it for you? I would prefer to leave it as-is...

Reply:
Step 1 is 27 seconds for me on a folder with ~800 duplicates. Step 2 was 50 seconds. I included the output from 2 runs below. The first is output into an empty directory with few if any duplicates and the step times are <1 second. The second run had 783 possible duplicate files to examine and required 77 seconds total to process. This is why I added the progress indication so the script would not apper to be frozen.

tldr;
My stats for the first pass over ~800 files
Copying all files to one folder...
(If you want, you can get them organized in folders based on year and month. Run with --divide-to-dates to do this)
100%|██████████████████████████████████████████████████████████████████████████████████████████████| 792/792 [01:22<00:00,  9.61it/s] 
=====================
=====================
Finding duplicates...
Step 1...
100%|████████████████████████████████████████████████████████████████████████████████████████████| 783/783 [00:00<00:00, 2365.13it/s]
Step 2...
100%|████████████████████████████████████████████████████████████████████████████████████████████████| 17/17 [00:00<00:00, 25.53it/s] 
Removing duplicates...
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:00<?, ?it/s] 
==================

Then the second pass over same files into already populated output directory

=====================
Copying all files to one folder...
(If you want, you can get them organized in folders based on year and month. Run with --divide-to-dates to do this)
100%|██████████████████████████████████████████████████████████████████████████████████████████████| 792/792 [01:22<00:00,  9.60it/s] 
=====================
=====================
Finding duplicates...
Step 1...
100%|██████████████████████████████████████████████████████████████████████████████████████████████| 783/783 [00:27<00:00, 28.27it/s]
Step 2...
100%|██████████████████████████████████████████████████████████████████████████████████████████████| 791/791 [00:50<00:00, 15.68it/s] 
Removing duplicates...
100%|████████████████████████████████████████████████████████████████████████████████████████████| 791/791 [00:00<00:00, 3682.25it/s] 
=====================